composer.json

```diff
@@ -30,6 +30,6 @@
         "mockery/mockery": "^0.9.5"
     },
     "scripts": {
-        "test": "phpunit"
+        "test": "phpunit --coverage-text"
```

Comment:
To enable code coverage reports you might have to re-enable XDebug by removing [this line](https://github.com/heiglandreas/JUnitDiff/blob/master/.travis.yml#L14) in the Travis CI configuration.